Python/Send_Supplies_Game/assets/note.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
What have you used this for @jeevanalok ??

Reply:
ohh? my bad! It was used to keep track of features that i am adding to the game. Not relevant to the PR. Am deleting it.